Makefile

```diff
@@ -2,6 +2,7 @@ default: docker-build
 
 setup:
 	(cd ui; yarn)
+	go get golang.org/x/tools/cmd/goimports
```

Comment:
Can we just run `go install` here as it's in the go mod require?

Reply:
Yes, fixed it.

Reply:
we actually need that `go get` because a `goimports` is not a package used by our application so it should not be a part of `go.mod`

Reply:
Why not be part of `go.mod`? It shouldn't add any extra bloat to the go binary right?

Reply:
No because `goimports` is a binary that you can install through any method. We are just using go to install it and hence technically this is not a part of our module. Also yeah we don't want unnecessary code as well!